integrations/tensorflow/python_projects/iree_tf/iree/tf/support/module_utils.py

```diff
@@ -0,0 +1,968 @@
+# Lint as: python3
```

Comment:
Is this copied from somewhere? Git isn't picking up the move/copy. Ditto with the below files

Reply:
Yes, it is just copied verbatim from bindings/python. Not sure what is up.